apps/webcert/src/faker/user/fakeUser.ts

```diff
@@ -16,5 +16,6 @@ export const fakeUser = (value?: Partial<User>): User => ({
   loginMethod: LoginMethod.FAKE,
   protectedPerson: false,
   careProviders: [fakeCareProvider({ id: unit.unitId, name: unit.unitName })],
+  origin: origin,
```

Comment:
Communicate condition through resource link as discussed so the origin does not reach the frontend